src/java/org/apache/cassandra/index/sai/disk/oldlucene/LuceneCompat.java

```diff
@@ -36,15 +38,31 @@ public static LongValues directReaderGetInstance(SeekingRandomAccessInput slice,
     {
         if (slice.order() == ByteOrder.LITTLE_ENDIAN)
             return DirectReader.getInstance(slice, bitsPerValue, offset);
-        // Lucene 7.5 and earlier used big-endian formatting
+        // Lucene 7.5 and earlier used big-endian ordering
         return LegacyDirectReader.getInstance(slice, bitsPerValue, offset);
```

Comment:
codestyle nit (very minor): in Cassandra we typically use ternary operator and single return in this case (highlights the symmetry of it; especially that the first if is not some error handling or anything unusual)